### PR TITLE
QUAD-122: membercomms

### PR DESCRIPTION
**Pull request by afromme**
These changes add pages for a member confirmation form and an email signup form, and embed those forms onto those pages.

### CODE REVIEW

**Review by jweiskoff — APPROVED** [review]
the script doesn't seem to work on the github page, but I'm assuming it did in testing. Assuming such, g2g from my end.

File: about/email_sign_up.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div id="muNBrVxMCA">
```
Comment (mhl10) [inline about/email_sign_up.md:1]:
```suggestion
Email Updates
======

<div id="muNBrVxMCA">
```

Comment (mhl10) [reply about/email_sign_up.md:1]: Above suggestion will look like this:

Screen Shot 2021-01-29 at 1 00 32 PM

File: about/membership_confirm.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<div id="dqRcPmjGnf">
```
Comment (mhl10) [inline about/membership_confirm.md:1]:
```suggestion
Membership Confirmation
======

<div id="dqRcPmjGnf">
```

**Review by bmaltzan — APPROVED** [review]
Looks good.

**Pull request merged**